test/library.spec.js

```diff
@@ -41,19 +41,6 @@ describe('createForm', () => {
     instance = getInstance();
   });
 
-  describe('config', () => {
```

Comment:
@subpx it'd be good to test here that `createForm` doesn't throw when `initialValues` is `undefined` for future safety, something like:

```javascript
   it('handles undefined initialValues', () => {
      const initialValues = undefined;
      const config = {initialValues, onSubmit: jest.fn()};

      expect(() => createForm(config)).not.toThrow();
    });
```

Reply:
OK I've added that test